supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/PipelinesApiAsyncBackendJobExecutionActor.scala

```diff
@@ -173,9 +173,42 @@ class PipelinesApiAsyncBackendJobExecutionActor(standardParams: StandardAsyncExe
 
   import mouse.all._
 
-  private def generateGcsLocalizationScript(inputs: List[PipelinesApiInput])(implicit gcsTransferConfiguration: GcsTransferConfiguration): String = {
-    val bundleFunction = (gcsLocalizationTransferBundle(gcsTransferConfiguration) _).tupled
-    generateGcsTransferScript(inputs, bundleFunction) |> bracketTransfersWithMessages("Localization")
+  private def generateGcsLocalizationScript(inputs: List[PipelinesApiInput], referenceFilesMapping: PipelinesApiReferenceFilesMapping)(implicit gcsTransferConfiguration: GcsTransferConfiguration): String = {
+    val referenceInputsToMountedPaths = PipelinesApiReferenceFilesMapping.getReferenceInputsToMountedPathMappings(referenceFilesMapping, inputs)
+    val referenceFilesLocalizationScript = {
+      val symlinkCreationCommands = referenceInputsToMountedPaths map {
+        case (input, absolutePathOnRefDisk) =>
+          s"mkdir -p ${input.containerPath.parent.pathAsString} && ln -s $absolutePathOnRefDisk ${input.containerPath.pathAsString}"
+      }
+
+      if (symlinkCreationCommands.nonEmpty) {
+        s"""
+           |# Faux-localizing reference files (if any) by creating symbolic links to the files located on the mounted reference disk
+           |${symlinkCreationCommands.mkString("\n")}
+           |""".stripMargin
+      } else {
+        ""
+      }
+    }
+
+    val regularFilesLocalizationScript = {
+      val regularFiles = inputs diff referenceInputsToMountedPaths.keySet.toList
```

Comment:
Yes this part is critical, not localizing files that were already fauxcalized. Not sure if that's something we can test for since I'm not exactly sure what that would look like...

Reply:
Presumably if the symlinking happens first you'd either get an error (which we'd detect) or override with a non-symlink (which we _are_ testing for) if you tried to full-localize a file over the top?

Reply:
Alternatively, we could do something similar to `papi_v2_log.test` which makes sure that there's no "localization" line for the reference file - only a "symlinking" line

Reply:
> Presumably if the symlinking happens first you'd either get an error (which we'd detect) or override with a non-symlink (which we _are_ testing for) if you tried to full-localize a file over the top?

Yes, `gsutil cp` would overwrite symlink with regular file